src/test/lsipii/User/StatusInfo.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+from enum import Enum
+
+from converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class StatusInfoValue(str, Enum):
+    SENT = ("SENT",)
+    PROCESSING = ("PROCESSING",)
+    WAITING_FOR_COMPLETION = ("WAITING_FOR_COMPLETION",)
+    READY = ("READY",)
+
+
+class StatusInfoResponse(CamelCaseModel):
+    status_label: str = Field(
+        ...,
+        title="Label of the status",
+        description="Label of the status in english",
```

Comment:
Languages are written with capital letters in English
```suggestion
        description="Label of the status in English",
```